app/javascript/packs/base.js

```diff
@@ -0,0 +1,9 @@
+import { setCurrentPageIconLink } from '../topNavigation/utilities';
```

Comment:
This is the beginning of a slow migration to pulling stuff out of base.js.erb. As mentioned up top, we're trying to avoid bloating base.js.erb as there is no code splitting for it.